canonical-web-and-design/jaas.ai
# Add /case-studies promo strip

## Done

Add /case-studies promo strip

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- Scroll to bottom of homepage and verify case studies promo works as expected

## Details

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/888
